src/test/java/org/eolang/jeo/representation/xmir/XmlNodeTest.java

```diff
@@ -93,6 +95,42 @@ void retrievesText() {
         );
     }
 
+    @Test
+    void retrievesChild() {
+        final XmlNode child = new XmlNode(
+            "<program><o>text</o></program>"
+        ).child("o");
+        final String expected = "<o>text</o>";
+        MatcherAssert.assertThat(
+            String.format(
+            "Retrieved XML: %s does not match with expected %s",
```

Comment:
@h1alexbel Something strange is happening with indentation here? Maybe we need two more spaces here?